.env.testing

```
@@ -35,6 +35,8 @@ QUEUE_CONNECTION=database
 CACHE_STORE=database
 # CACHE_PREFIX=
 
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)